docs/design/diagrams/README.md

```diff
@@ -0,0 +1,42 @@
+# Diagrams of Armada Architecture
+
+## Systems Diagram
+
+![Systems Diagram](armada-system.png)
+
+This diagram shows the high-level architecture of Armada. It is made of 4 parts:
+
+### Armada Server Components
+
+These are the components that control the operation of the different executors across different k8s clusters (See https://armadaproject.io/design)
+
+It includes:
+- **Armada Server**: The main component of Armada. It is responsible for the scheduling of jobs and the management of the executors.
```

Comment:
Not sure if Armada Server is responsible for the management of the executors.  It can communicate with them but management means to me orchestrating.  I think the executors are independent components on kubernetes clusters and they can communicate to the server.